codelists/views.py

```diff
@@ -113,6 +112,109 @@ def get_signoff_formset(self, data=None, files=None):
         return self.SignOffFormSet(data, files, prefix="signoff")
 
 
+@method_decorator(login_required, name="dispatch")
+class CodelistUpdate(TemplateView):
+    ReferenceFormSet = modelformset_factory(
+        Reference, form=ReferenceForm, can_delete=True
+    )
+    SignOffFormSet = modelformset_factory(SignOff, form=SignOffForm, can_delete=True)
```

Comment:
Do these definitions belong in `forms.py`?

Reply:
I hadn't even considered putting them in there, good call!

Reply:
Fixed in 84b5596